homework/setup.sh

```diff
@@ -0,0 +1,23 @@
+# Run the docker container with Postgres
```

Comment:
Very cool script! I copy what I wrote on Vivek's request, which I think you will also enjoy: 

I really like that you created a bash script to automate everything you were doing. 

However, it's hard for me to imagine how this might be used! 

I think a more "realistic" polish along the lines that you are thinking might go like this: 

1. Assume there is a database running somewhere but that the user of your script wants to tell you where it is and how to connect to it. 

2. Assume there are CSV files to be loaded, but the users wants to tell you where they are. 

Then, I would make the postgres connection info and the csv location CLI arguments to your python script. Thus, you have a python script that can be run from anywhere, getting the csv's (in the prescribed format, of course) and loading them into the database. 

Next is to "wrap up" the requirements (pyscopg2). Two common ways to do this: 

1. Print a "requirements.txt" file from pip (google it, but you just use `pip freeze > requirements.txt`. Then, I would have this "bash script" run the following: 

```sh
python3 -m venv venv && source venv/bin/activate
pip install -r requirements.txt
python week1.py $1 $2
```

2. You can create a Dockerfile and make your little "data loading" application a Docker container! The Dockerfile would look like this: 

```dockerfile
FROM python:3.7

RUN mkdir -p /usr/src/app
WORKDIR /usr/src/app

COPY requirements.txt .
RUN pip install --no-cache-dir -r requirements.txt

COPY week1.py /usr/src/app/week1.py

CMD ["python", "week1.py"]
```

And then you can build it into a docker image (vivek/week1) and run it with `docker run vivek/week1` -- then you just need to deal with mounting the folders with the csv's and pointing it to the csv's (or maybe you get them from a remote file system, like S3!). 

Anyways, just to give you some food for thought ;)